### PR TITLE
Add reporting chapter

### PR DESCRIPTION
**Pull request by pat-s**
Will appear as "chapter 6" in the book since we're indexing at 0.

content is quite dense so far but will also have a practical part after the theory.

### CODE REVIEW

**Comment by krlmlr** [comment]
Looks good. What does the conflict mean?

File: book/05-reporting.Rmd [hunk 1]
```diff
@@ -1 +1,164 @@
...
+
+To insert code into an R Markdown document, one needs to add a so called "code chunk".
+
+````
+```{<language>}
+
+```
+````
+
+This tells the document that everything within the three backticks should be interpreted as code using the given language.
```
Comment (krlmlr) [inline book/05-reporting.Rmd:115]: Would this be easier to grasp if we used a "white lie" that the chunk header is of the form ````` ```{r} ````` ?

File: book/05-reporting.Rmd [hunk 2]
```diff
@@ -1 +1,164 @@
...
+
+- Talks using presentation slides ([xaringan](https://bookdown.org/yihui/rmarkdown/xaringan.html), [ioslides](https://bookdown.org/yihui/rmarkdown/ioslides-presentation.html), [Slidy](https://bookdown.org/yihui/rmarkdown/slidy-presentation.html), [Beamer](https://bookdown.org/yihui/rmarkdown/beamer-presentation.html), [Powerpoint](https://bookdown.org/yihui/rmarkdown/powerpoint-presentation.html))
+- written reports (Word, PDF), possibly using LaTeX input
+
+R objects (containing results) can directly be used in the reports to present the results (data, plots).
+If the complete workflow of an analysis has been setup in R, changes at certain stages of the workflow (e.g. incoming data) can easily be integrated.
+
+This is the point where packages like [drake](https://docs.ropensci.org/drake/), [workflowr](https://jdblischak.github.io/workflowr/) and [rrtools](https://github.com/benmarwick/rrtools) jump in to simplify reproducible workflows in R.
+
+A widely used concept is to start a project following the structure of an R package.
```
Comment (krlmlr) [inline book/05-reporting.Rmd:152]: An existing section is dedicated to this.

**Review by krlmlr — COMMENTED** [review]
Thanks, looks good.

File: book/05-reporting.Rmd (hunk 1 above)
Comment (pat-s) [reply book/05-reporting.Rmd:115]: Umm - I'd say it's fine as it is.
Such things can fire back at some point - maybe not at us but somebody else.

File: book/05-reporting.Rmd (hunk 2 above)
Comment (pat-s) [reply book/05-reporting.Rmd:152]: I currently do not see one (neither in the deployed book nor in the existing files).

If you tell me, I can add a cross-ref.

**Comment by pat-s** [comment]
@krlmlr ready to merge?

**Pull request merged**